R/scan.R

```diff
@@ -0,0 +1,74 @@
+#' Read a scan results text file. scan.read returns a 1 segment dataframe.
+#' Reads in scan results and puts them in the same format as the output
+#' of dwt.read. See 'dwt', and 'segment' for more information.
+#' 
+#' Data is in seconds.
+#' @param fn, the file name to read from.
+#' @param separating_factor In lieu of a known time between segments, 
+#'        seperate with a multple of the longest dwell.
+#' @return A list of bursts.
+#' @examples
+#' \dontrun{
```

Comment:
We should make all the examples functional, and we should include a "filename.scan" example file so that they have a working example file to look at and test. (This file would go in `inst/extdata`)